packages/gravity-ui-web/gulp/color-scheme-tables.js

```diff
@@ -0,0 +1,66 @@
+const Color = require('color');
+const kebabCase = require('lodash.kebabcase');
+const { colorSchemes } = require('@buildit/gravity-particles');
+
+
+
```

Comment:
We linting our JS files? Should turn on something to prevent multiple lines of whitespace.

Reply:
We don't currently lint our JS in this package. But, I agree that we should. I'll add eslint.